cmd/tcpgoon.go

```diff
@@ -63,6 +65,13 @@ func validateRequiredArgs(params *tcpgoonParams, args []string) error {
 		return errors.New("Number of required parameters doesn't match")
 	}
 	params.hostPtr = args[0]
+
+	addrs, err := net.LookupIP(params.hostPtr)
+	if err != nil {
+		return errors.New("Domain name not resolvable")
+	}
+	params.hostPtr = addrs[0].String()
```

Comment:
is there a corner case where you may not get an err but the slice is empty? Should we check then the slice size before checking the first element?

Reply:
I don't think so, but we can add this check